packages/styles/package.json

```diff
@@ -21,7 +21,7 @@
     "lint": "stylelint 'src/**/*.scss'",
     "serve": "serve -p 5000",
     "test:watch": "echo 'No tests available yet.'",
-    "test": "node src/test.js"
+    "test": "npm run build && node src/test.js"
```

Comment:
Discussion here:

we can force the build here or create a separated step in the circleCI config.

if we create a separated step the test output wont containg build traces (red mark):


<img width="665" alt="screen shot 2018-07-19 at 12 39 21" src="https://user-images.githubusercontent.com/2657897/42937931-db286616-8b50-11e8-9714-66a9e499d9fb.png">

Reply:
That seems reasonable, I'll create a different step in the job